src/cli.ts

```diff
@@ -77,6 +77,28 @@ export default class CLI {
 				process.exit(1);
 			}
 		});
+
+		const helpEvent = this.keyListener.bindEvent({
+			keyName: ['h'],
+			handler: function (keyName) {
+				console.log(`
+Start => Earth(0, 0)
+Goal => Moon(0, 250)
+
+Once Launched, the Ship moves Forward 
+
+W -> Increase Forward Speed
+S -> Decrease Forward Speed 
+A -> Move Left
+D -> Move Right
+Max Speed - 5 speedunit
+Min Speed - 1 speedunit (after leaving (0, 0))
+
+Press H -> Show the Ship Reference Manual
```

Comment:
LGTM :rocket: 

@ish-u  Could you add a message [here](https://github.com/pyymenta/spacecraft-cli/blob/master/src/cli.ts#L86) with the instructions to access the help screen?

Reply:
Ok i will do that today

Reply:
I have added the instruction to access Help Screen.